website/docs/dictformats.md

```diff
@@ -70,7 +70,7 @@ Main file of DictD dictionary (.dict) or Xdxf dictionary (.xdxf) can be compress
 
 ### Slob
 
-GoldenDict can render formulas in TeX format via Mimetex program. Just place "mimetex.cgi" file beside GoldenDict executable file.
+GoldenDict can render formulas in TeX format with the built-in web engine.
```

Comment:
```suggestion
GoldenDict can render TeX formulas when a slob dictionary bundles MathJax.
```
Just to be more precise.

All slob dicts with raw TeX comes from here, which bundles mathjax (since 10 years ago 😅 ).
https://github.com/itkach/mw2slob/tree/56da4744af4a6b9099e6bde9bad6cd30ae802226/mwscrape2slob/_MathJax